### ISSUE TITLE
Notifications unread status not being sync

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#2235]
### Problem

On Desktop on the GIG group, all notifications are read, but on my phone on the PWA, these are not only unread, but clicking "Mark as read" doesn't do anything:

![Screenshot_20240719-154053](https://github.com/user-attachments/assets/0c17bf6a-3209-4631-bccb-4e7d52c8de4d)

![Screenshot_20240719-154051](https://github.com/user-attachments/assets/f2017b67-3233-440b-96ac-4abefe6b9b3d)

EDIT: what's odd is that this works on the testing website...

EDIT2: OK, after @Silver-IT joined the group, suddenly the notifications looked like this:

![tmp-1721429497316](https://github.com/user-attachments/assets/a5918768-e21e-46d5-bac3-b3ef8396d7d0)

In other words:

- The two notifications that I had clicked "Mark as read" on PWA that wouldn't mark themselves as read, suddenly became read.
- The new notification came in as unread
- And this time I was *was* able to click "Mark as read" and it did sync

### Solution

Fix.

**Comment by dotmacro** [comment]
I just encountered this issue again on desktop. Using two different devices:
1. (already signed in and using app on device1)
2. Sign in on device2. The notifications that were marked as read before I signed in on device2 remained marked as read on both.
3. Mark new notification as read on device2. (Same new notification is not automatically marked as read on device1.)
4. `Mark as read` does not work on device1.
5. Refreshing page does not help, nor hard refresh. Restarting browser does not help. Restarting device does not help.
6. Log out device1 and log back in. 
7. Notification now automatically marked as read.

**Comment by Silver-IT** [comment]
Having tested several times on my side, I can not encounter this issue. 🤔
